Fuzz TestParseData failures on nacl

I'm trying to update the vendored pprof in the Go repository, and the `nacl-386` and `nacl-amd64p32` are failing with the following error:

```
--- FAIL: TestParseData (0.37s)
	fuzz_test.go:14: Problem reading directory testdata/ : open testdata/: No such file or directory
FAIL
FAIL	cmd/vendor/github.com/google/pprof/fuzz	0.443s
2017/08/20 11:28:07 Failed: exit status 1
```

All the other trybots are fine.

The vendored pprof commit is 2191094077cac7cd05ced16c0768a2e57fe5371d.

[Full nacl-386 log](https://storage.googleapis.com/go-build-log/e0643842/nacl-386_5da3967c.log)
[Link the Go change](http://golang.org./cl/57370)

--- a/fuzz/fuzz_test.go
+++ b/fuzz/fuzz_test.go
@@ -3,9 +3,9 @@ package pprof
 import (
 	"io/ioutil"
 	"testing"
+	"runtime"
 
 	"github.com/google/pprof/profile"
-	"runtime"
 )
 
 func TestParseData(t *testing.T) {

--- a/fuzz/fuzz_test.go
+++ b/fuzz/fuzz_test.go
@@ -5,9 +5,14 @@ import (
 	"testing"
 
 	"github.com/google/pprof/profile"
+	"runtime"
 )
 
 func TestParseData(t *testing.T) {
+	if runtime.GOOS == "nacl" {
+		t.Skip("no direct filesystem access on Nacl")
+	}
+
 	const path = "testdata/"
 	files, err := ioutil.ReadDir(path)
 	if err != nil {

--- a/fuzz/fuzz_test.go
+++ b/fuzz/fuzz_test.go
@@ -2,8 +2,8 @@ package pprof
 
 import (
 	"io/ioutil"
-	"testing"
 	"runtime"
+	"testing"
 
 	"github.com/google/pprof/profile"
 )

